### PR TITLE
[SIN-20] update location, dates, registration and sponsors

### PR DESCRIPTION
**Pull request by sergiubodiu**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

### CODE REVIEW

File: content/events/2020-singapore/contact.md [hunk 1]
```diff
@@ -19,5 +19,3 @@ Big THANK YOU to [Wenshu Kwek](https://twitter.com/wenshukwek) for designing the
 
 {{< list_organizers >}}
 
-{{< list_core >}}
```
Comment (bridgetkromhout) [inline content/events/2020-singapore/contact.md]: The "{{< list_core >}}" item is not optional here. It links to the global team. As you don't have this PR set for us to edit it, I'll wait for you to restore this line.

File: data/events/2020-singapore.yml [hunk 2]
```diff
@@ -3,44 +3,44 @@ year: "2020" # The year of the event. Make sure it is in quotes.
...
+cfp_date_end: 2020-02-09T23:59:59+08:00
 cfp_date_announce: 2020-02-15T23:59:59+08:00
 
 cfp_open: "true"
-cfp_link: "" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+cfp_link: "https://www.papercall.io/dodsin20" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
 
 registration_date_start: # start accepting registration. Leave blank if registration is not open yet
-registration_date_end: # close registration. Leave blank if registration is not open yet.
+registration_date_end: "https://ti.to/devopsdays-singapore/2020/" # close registration. Leave blank if registration is not open yet.
```
Comment (bridgetkromhout) [inline data/events/2020-singapore.yml]: This should go in "registration_link:" not in a date field.

File: data/events/2020-singapore.yml [hunk 3]
```diff
@@ -63,24 +63,28 @@ team_members: # Name is the only required field for team members.
     bio: "Abhishek Chowdhury is the lead DevOps Engineer at Credit Suisse APAC IT."
     image: "abhishek-chowdhury.jpg"
 
-  - name: "Vincent De Smet"
```
Comment (bridgetkromhout) [inline data/events/2020-singapore.yml]: Is this organizer stepping down for this year? If so we will also remove him from Slack and email.

**Comment by mattstratton** [comment]
Note: in the future, please set your PR to allow edits from maintainers; this will definitely accelerate the merging of PRs:

https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

File: content/events/2020-singapore/contact.md (hunk 1 above)
Comment (sergiubodiu) [reply content/events/2020-singapore/contact.md]: made the changes

File: data/events/2020-singapore.yml (hunk 2 above)
Comment (sergiubodiu) [reply data/events/2020-singapore.yml]: updated the registration_link and the start / end dates

File: data/events/2020-singapore.yml (hunk 3 above)
Comment (sergiubodiu) [reply data/events/2020-singapore.yml]: Yes, unfortunately he is not able to contribute this year, you can remove him from Slack and email

**Comment by dianabodiu** [comment]
> Note: in the future, please set your PR to allow edits from maintainers; this will definitely accelerate the merging of PRs:
> 
> https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Thanks for the guide  will do

File: data/events/2020-singapore.yml (hunk 3 above)
Comment (kmugrage) [reply data/events/2020-singapore.yml]: Vincent removed from Slack and mailing list.

**Pull request merged**